graphql_compiler/compiler/expressions.py

```diff
@@ -940,21 +947,29 @@ def to_gremlin(self):
         intersects_operator_format = u'(!{left}.{operator}({right}).empty)'
         negated_dotted_operator_format = u'!{left}.{operator}({right})'
 
-        translation_table = {
-            u'=': (u'==', immediate_operator_format),
-            u'!=': (u'!=', immediate_operator_format),
-            u'>=': (u'>=', immediate_operator_format),
-            u'<=': (u'<=', immediate_operator_format),
-            u'>': (u'>', immediate_operator_format),
-            u'<': (u'<', immediate_operator_format),
-            u'+': (u'+', immediate_operator_format),
-            u'||': (u'||', immediate_operator_format),
-            u'&&': (u'&&', immediate_operator_format),
-            u'contains': (u'contains', dotted_operator_format),
-            u'not_contains': (u'contains', negated_dotted_operator_format),
-            u'intersects': (u'intersect', intersects_operator_format),
-            u'has_substring': (u'contains', dotted_operator_format),
-        }
+        # Comparing null to a value does not make sense.
+        if self.left == NullLiteral:
+            raise AssertionError(
+                u'The left expression cannot be a NullLiteral! Received operator '
+                u'{} and right expression {}.'.format(self.operator, self.right))
+        else:
```

Comment:
nit: since the `raise` breaks control flow already, I'd remove the `else` here since I feel that might make the code more readable – and it will definitely make Git not show these lines as modified.